apps/constants/base.py

```diff
@@ -5,20 +5,19 @@
 
 
 # Add-on and File statuses.
-STATUS_NULL = 0
-STATUS_UNREVIEWED = 1
-STATUS_PENDING = 2
-STATUS_NOMINATED = 3
-STATUS_PUBLIC = 4
-STATUS_DISABLED = 5
+STATUS_NULL = 0  # No review type chosen yet, add-on is incomplete.
+STATUS_UNREVIEWED = 1  # Waiting for prelim review.
+STATUS_PENDING = 2  # Personas (lightweight themes) waiting for review.
+STATUS_NOMINATED = 3  # Waiting for full review.
+STATUS_PUBLIC = 4  # Fully reviewed.
```

Comment:
I could add "or pending update", but then it would need this explanation too. And then I would need to duplicate it for the `STATUS_LITE` below, because it's the same. I don't mind doing that, but from a code point of view, "pending updates" is just a small/limited "trick" to display some addons in a specific queue in the editor tools. It's not a status in itself.